pkg/apis/apis/v1alpha1/types.go

```diff
@@ -350,6 +354,18 @@ type Identity struct {
 	SecretRef *corev1.SecretReference `json:"secretRef"`
 }
 
+// APIExportPolicy is a wrapper type around the multiple options that would be allowed.
+type APIExportPolicy struct {
```

Comment:
this needs a yaml-patch file to enforce at least one of the variants (here only one), compare https://github.com/kcp-dev/kcp/blob/main/config/crds/apis.kcp.dev_apibindings.yaml-patch

Reply:
Is this generated or do I have to make it manually? 

"enforce the variants" can you explain this to me offline, because I don't know if I follow or understand.

Reply:
Got an explanation from Andy and now I understand and have added.